### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Lcharvol**

### CODE REVIEW

File: src/client/App.test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './component/App';
```
Comment (eric-basley) [inline src/client/App.test.js]: Should be components instead of component

File: src/client/component/Container/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from './Container';
```
Comment (eric-basley) [inline src/client/component/Container/index.js]: file should be renamed index.js

File: src/client/component/Container/Container.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
Comment (eric-basley) [inline src/client/component/Container/Container.js]: rename file index.js

File: src/client/component/Dataflow/Dataflow.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.dataflowcontainer
+{
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  width:100%;
+  min-height: 220px;
+}
+
+.dataflowelem
```
Comment (eric-basley) [inline src/client/component/Dataflow/Dataflow.css]: wrong class name dataflowelm does not exist anymore

**Comment by Lcharvol** [comment]
Conflicts should be fixed.

File: src/client/components/App/index.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+class App extends Component {
+  state = {
+    showSidePanel: true,
+  }
+
+  displayShowPanel = () => {
+    const { showSidePanel } = this.state;
+
+    this.setState({ showSidePanel: !showSidePanel });
```
Comment (eric-basley) [inline src/client/components/App/index.js]: Utiliser setState(state =>  ({ showSidePanel: !state.showSidePanel }));

File: package.json [hunk 6]
```diff
@@ -15,6 +15,8 @@
     "Faceted Search"
   ],
   "dependencies": {
+    "font-awesome": "^4.7.0",
```
Comment (eric-basley) [inline package.json]: Pourquoi charger un npm alors que fa est chargé par index.html

File: src/client/components/App/index.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  render() {
+    const { showSidePanel } = this.state;
+    return (
+      <div className="App">
+        <SidePanel
+          showSidePanel={showSidePanel ? 'hidden' : ''}
```
Comment (eric-basley) [inline src/client/components/App/index.js]: utiliser un boolan

File: src/client/components/Header/index.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+      <LanguageSelector />
+    </div>
+  </nav>
+);
+
+Header.propTypes = {
+  title: PropTypes.string.isRequired,
+};
+
+export default connect(Header);
```
Comment (eric-basley) [inline src/client/components/Header/index.js]: Header must not be a container, only App in your context should be one

**Pull request merged**